images/linux/scripts/installers/oc.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+################################################################################
+##  File:  aws.sh
+##  Desc:  Installs the AWS CLI
```

Comment:
This block looks like copy-paste from AWS CLI

Reply:
changed :)